<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>109</y>
      <width>901</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QTreeWidget" name="TreeResult">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QTreeWidget" name="CompareResult">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="DirectoryButton">
       <property name="text">
        <string>Directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SearchButton">
       <property name="text">
        <string>Start search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CompareButton">
       <property name="text">
        <string>Compare</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopButton">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>421</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="0">
      <widget class="QLabel" name="FilesLabel">
       <property name="text">
        <string>Files:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="DurationLabel">
       <property name="text">
        <string>Duration:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QProgressBar" name="ProgressBar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="maximum">
        <number>0</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="SourceFolderLabel">
       <property name="text">
        <string>Source Folder:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="ProgressLabel">
       <property name="text">
        <string>Progress:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="SourceFolder">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="Duration">
       <property name="text">
        <string>00::00::00</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="Files">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="TopBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="Menu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="Deleteaction"/>
    <addaction name="Infoaction"/>
    <addaction name="Aboutaction"/>
    <addaction name="Exitaction"/>
   </widget>
   <addaction name="Menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="Deleteaction">
   <property name="text">
    <string>Delete selected</string>
   </property>
  </action>
  <action name="Aboutaction">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="Exitaction">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="Infoaction">
   <property name="text">
    <string>Program info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Exitaction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotExit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Aboutaction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Infoaction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotInfo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Deleteaction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>slotDeleteselected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>325</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotExit()</slot>
  <slot>slotInfo()</slot>
  <slot>slotAbout()</slot>
  <slot>slotDeleteselected()</slot>
 </slots>
</ui>
